apps/nextjs-website/react-components/components/MegaHeader/MegaHeader.Helpers.ts

```diff
@@ -247,3 +247,15 @@ export const Overlay = styled('div')({
     display: 'block',
   },
 });
+
+export const NewBadge = styled('span')({
```

Comment:
```suggestion
export const LinkLabel = styled('span')({
```
The component accepts any string, so no reason to call it "New" (it's also confusing, is there an OldBadge?)

Let's call it label instead of badge to be consistent with the design team's terminology.